src/Resources/snippet/de_DE/storefront.de-DE.json

```
@@ -0,0 +1,5 @@
+{
+  "cookie": {
+    "sgSaveCustomerData": "Formulardaten Reservierung"
```

I may have the context wrong, I'd think that it should be one word though. Correct me if I'm wrong.

```suggestion
    "sgSaveCustomerData": "Reservierungsformulardaten"
```